tests/test_csc.py

```diff
@@ -114,6 +115,7 @@ async def test_initial_info(self):
                     "logMessage",
                     "settingVersions",
                     "settingsApplied",  # not yet supported by salobj
+                    "largeFileObjectAvailable",  # not supported by CSC
```

Comment:
This is because largeFileObjectAvailable is now in generics, but the new code to hide it from most CSCs isn't yet in ts_sal? It is most unfortunate that all salobj-based CSCs are now going to claim to support this event for awhile.